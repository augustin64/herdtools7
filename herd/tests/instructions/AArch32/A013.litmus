../ARM/A013.litmus