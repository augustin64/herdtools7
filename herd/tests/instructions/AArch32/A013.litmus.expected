../ARM/A013.litmus.expected